pkg/release/release.go

```diff
@@ -121,6 +121,10 @@ func New(config Config) (*Release, error) {
 	return r, nil
 }
 
+func (r *Release) Condition() ConditionSet {
+	return r.condition
+}
```

Comment:
I do not follow this here.

Reply:
So we can do `release.Install(...., release.Condition().Secret(ctx, namespace, name))`.

Reply:
I see. This makes sense. Just to be clear. `release` in your example is not the package but an instance of the service implementation, right?

Reply:
Yeah.